frontend/src/routes/index.js

```diff
@@ -58,7 +58,7 @@ export const AppRouter = () => {
         <PrivateRoute path="/clients"      component={ClientsPage} roles={['admin']} />
         <PrivateRoute path="/time_entries" component={TimeEntriesPage} />
         <PrivateRoute path="/events/:id/edit"  component={EditEventTemplatePage} roles={['admin']} />
-        <PrivateRoute path="/events/new"       component={NewEventTemplatePage} roles={['admin']}/>
+        <PrivateRoute path="/events/new"       component={NewEventTemplatePage}/>
```

Comment:
why did u remove admin roles?

Reply:
only an admin can create an event template?

Reply:
the user can create a personal event for himself

Reply:
yes but you have 2 endpoint
 - event template
 - user event
So i think you have to add a new path

Reply:
Yeah you're right I close this PR for the moment and I create a new one when we take the feature for this case.